### ISSUE TITLE
train_annotations.json is missing

### ISSUE THREAD

**Issue by BAILOOL** [visipedia/iwildcam_comp#1]
"Train and validation annotations, and test information 4.8MB" includes the following files:
 - ```test_annotations.json```
 - ```test_information.json```
 - ```val_annotations.json```

Apparently, ```train_annotations.json``` is missing.

**Comment by macaodha** [comment]
Thank's for flagging this. I think I uploaded old annotation files. I will fix this tomorrow.

**Comment by macaodha** [comment]
Train annotations are fixed. Test set is still work in progress, and will be online soon.

**Comment by macaodha** [comment]
Just to be sure I am currently preparing a a new version of all the files for download. I'll close this issue when they are uploaded.

**Comment by macaodha** [comment]
The new dataset is uploaded. Thanks for your patience.

**Comment by BAILOOL** [comment]
While my ```mdsum``` check works fine for all files, I have discovered about 2978 images missing from the training dataset, 391 images from validation, and 16740 from test dataset. Missing files can be found [here](https://drive.google.com/open?id=1IhtyASl9nHn1r3QC-IIbrWjK3uteOjTL).

Do we simply proceed without them?

**Comment by macaodha** [comment]
I re-downloaded the small version of the dataset from the links provided to double check. The md5sums were the same as listed. I was able to extract all the images, including the ones you listed. Perhaps something went wrong in the unzipping stage for you? Let me know if you are able find everything.

**Comment by BAILOOL** [comment]
@macaodha, yes indeed, there was a problem with unzipping. Thank you for your support!
